core/src/OpenMPTarget/Kokkos_OpenMPTarget_Instance.hpp

```diff
@@ -0,0 +1,40 @@
+#include <Kokkos_OpenMPTarget.hpp>
+
+namespace Kokkos {
+namespace Experimental {
+namespace Impl {
+
+class OpenMPTargetInternal {
+ private:
+  OpenMPTargetInternal();
+  OpenMPTargetInternal(const OpenMPTargetInternal&) = default;
+  OpenMPTargetInternal& operator=(const OpenMPTargetInternal&) = default;
+
+ public:
+  void fence();
+
+  /** \brief  Return the maximum amount of concurrency.  */
```

Comment:
This comment has a different style than the others in the file.

Reply:
leftover style from doxygen. I leave it this way for now in case we ever try doxygen again